benchmark/CMakeLists.txt

```diff
@@ -9,17 +9,47 @@ if(BENCHMARK)
     	GIT_TAG origin/main
 	)
 
-	FetchContent_MakeAvailable(googlebenchmark)
+	FetchContent_Declare(
+    openssl
+    GIT_REPOSITORY https://github.com/janbar/openssl-cmake.git
+    GIT_TAG origin/master
+    OPTIONS "WITH_APPS OFF"
+    )
+
+    FetchContent_Declare(
+    zlib
+    GIT_REPOSITORY https://github.com/madler/zlib.git
+    GIT_TAG v1.2.13
+    OPTIONS "CMAKE_POSITION_INDEPENDENT_CODE True"
+    )
```

Comment:
Fetch both openssl and zlib.

openssl will be removed soon: https://github.com/ZigRazor/CXXGraph/issues/274